index.js

```diff
@@ -1,8 +1,17 @@
 'use strict';
+
```

Comment:
Can you please not change unrelated things? This was intentionally not there. :wink: